nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -469,49 +469,103 @@ def test_forward_multi_input():
+        with tf.Session() as sess:
+            graph_def = tf.graph_util.convert_variables_to_constants(
+                sess,
+                sess.graph.as_graph_def(add_shapes=True),
+                ['ResizeBilinear'],
+                )
+
+            tf_output = run_tf_graph(sess, data,
+                    'Const:0', 'ResizeBilinear:0')
+
+            tvm_output = run_tvm_graph(graph_def,
+                                       data,
+                                       "Const", tf_output.shape, data.dtype)
+
+            np.testing.assert_allclose(tf_output, tvm_output, atol=1e-3, rtol=1e-3)
+
+            sess.close()
+
+def test_forward_resize_bilinear():
+    """ Resize Bilinear """
+
+    _test_resize_bilinear((4, 16, 32, 32), [50, 50], False)
+    _test_resize_bilinear((6, 32, 64, 64), [20, 20], True)
+
+
 #######################################################################
 # Inception V3
 # ------------
 def test_forward_inception_v3():
     '''test inception V3 model'''
     with tf.Graph().as_default():
-        (data, graph_def) = nnvm.testing.tf.get_workload_inception_v3()
+        graph_def = nnvm.testing.tf.get_workload('InceptionV3/inception_v3_2016_08_28_frozen-with_shapes.pb')
         # Call the utility to import the graph definition into default graph.
         graph_def = nnvm.testing.tf.ProcessGraphDefParam(graph_def)
 
-        tvm_output = run_tvm_graph(graph_def, data, 'input', (1, 1001), 'float32')
+        data = np.random.uniform(size=(1, 299, 299, 3)).astype('float32')
+
         with tf.Session() as sess:
             tf_output = run_tf_graph(sess, data, 'input:0', 'InceptionV3/Predictions/Reshape_1:0')
-
-            top_tvm = np.squeeze(tvm_output).argsort()[-3:][::-1]
-            top_tf = np.squeeze(tf_output).argsort()[-3:][::-1]
-
-            np.testing.assert_allclose(top_tf, top_tvm, rtol=1e-5, atol=1e-5)
+            tvm_output = run_tvm_graph(graph_def, data, 'input', tf_output.shape, 'float32')
+            np.testing.assert_allclose(tf_output, tvm_output, rtol=1e-5, atol=1e-5)
 
 #######################################################################
 # Inception V1
 # ------------
 def test_forward_inception_v1():
     '''test inception V1 model'''
     with tf.Graph().as_default():
-        (data, tvm_data, graph_def) = nnvm.testing.tf.get_workload_inception_v1()
+        graph_def = nnvm.testing.tf.get_workload("InceptionV1/classify_image_graph_def-with_shapes.pb")
         # Call the utility to import the graph definition into default graph.
         graph_def = nnvm.testing.tf.ProcessGraphDefParam(graph_def)
 
-        tvm_output = run_tvm_graph(graph_def, tvm_data, 'DecodeJpeg/contents', (1, 1008), 'float32')
+        # Build an image from random data.
+        from PIL import Image
+        img_array = np.random.uniform(size=(1, 600, 600, 3)).astype("uint8")
+        img = Image.frombuffer('RGB', (600, 600), img_array.tostring(), 'raw', 'RGB', 0, 1)
+        img.save('tf_test.jpg');
```

Comment:
not sure why you need to save an image, but shouldn't this be deleted after test is done?
Or it may be better to use temporary file here.

Reply:
@masahi  used temp for image. Please review now.